tests/unit/endpoints/test_image_edit_endpoint.py

```
@@ -0,0 +1,381 @@
+        assert payload["seed"] == 42
+
+    def test_format_payload_extra_inputs_cannot_overwrite_reserved(self):
+        """Reserved keys (prompt, image, url, mask) are protected from --extra-inputs."""
+        model_endpoint = create_model_endpoint(
+            EndpointType.IMAGE_EDIT,
+            model_name="black-forest-labs/FLUX.2-klein-4B",
+            extra=[
+                ("prompt", "HIJACKED"),
+                ("image", "not-a-file"),
+                ("url", "https://malicious.example"),
+                ("mask", "/tmp/mask.png"),
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/mask.png"

(S108)